modules/config-custom-ssl-certs-manual.adoc

```diff
@@ -1,53 +1,95 @@
-[[config-custom-ssl-certs-manual]]
-= Adding TLS Certificates to the {productname} Container
+[id="config-custom-ssl-certs-manual"]
+= Adding additional Certificate Authorities to the {productname} container
 
-To add custom TLS certificates to {productname}, create a new
-directory named `extra_ca_certs/` beneath the {productname} config
-directory. Copy any required site-specific TLS certificates to this new
-directory.
+The `extra_ca_certs` directory is the directory where additional Certificate Authorities (CAs) can be stored to extend the set of trusted certificates. These certificates are used by {productname} to verify SSL/TLS connections with external services. When deploying {productname}, you can place the necessary CAs in this directory to ensure that connections to services like LDAP, OIDC, and storage systems are properly secured and validated.
 
-[[add-certificates-to-quay-container]]
-== Add TLS certificates to {productname}
-. View certificate to be added to the container
+For standalone {productname} deployments, you must manually create this directory and copy the additional CA certificates into that directory. 
+
+.Prerequisites
+
+* You have a CA for the desired service.
+
+.Procedure
+
+. View the certificate to be added to the container by entering the following command:
 +
-```
+[source,terminal]
+----
 $ cat storage.crt
+----
++
+.Example output
++
+[source,terminal]
+----
 -----BEGIN CERTIFICATE-----
-MIIDTTCCAjWgAwIBAgIJAMVr9ngjJhzbMA0GCSqGSIb3DQEBCwUAMD0xCzAJBgNV
-[...]
+MIIDTTCCAjWgAwIBAgIJAMVr9ngjJhzbMA0GCSqGSIb3DQEBCwUAMD0xCzAJBgNV...
 -----END CERTIFICATE-----
-```
+----
+
+. Create the `extra_ca_certs` in the `/config` folder of your {productname} directory by entering the following command:
++
+[source,terminal]
+----
+$ mkdir -p /path/to/quay_config_folder/extra_ca_certs
+----
 
-. Create certs directory and copy certificate there
+. Copy the CA file to the `extra_ca_certs` folder. For example:
 +
-```
-$ mkdir -p quay/config/extra_ca_certs
-$ cp storage.crt quay/config/extra_ca_certs/
-$ tree quay/config/
-├── config.yaml
-├── extra_ca_certs
-│   ├── storage.crt
-```
+[source,terminal]
+----
+$ cp storage.crt /path/to/quay/config/extra_ca_certs/
```

Comment:
I see `/path/to/quay/config/extra_ca_certs/` and  `/path/to/quay_config_folder/extra_ca_certs`, suggest to use same one